labgrid/driver/usbvideodriver.py

```diff
@@ -34,12 +40,21 @@ def select_caps(self, hint=None):
         raise InvalidConfigError("Unkown video format {} for device {:04x}:{:04x}".format(
             variant, self.video.vendor_id, self.video.model_id))
 
+    def get_pipeline(self):
+        match = (self.video.vendor_id, self.video.model_id)
+        if match == (0x046d, 0x082d):
+            return "v4l2src device={} ! {} ! h264parse ! fdsink"
+        if match == (0x046d, 0x0892):
+            return "v4l2src device={} ! {} ! jpegparse ! vaapijpegdec ! videoconvert ! vaapipostproc ! vaapih264enc ! h264parse ! fdsink"
```

Comment:
According to @michaelolbrich, the following should work as well and would be preferred.

```suggestion
            return "v4l2src device={} ! {} ! decodebin ! vaapipostproc ! vaapih264enc ! h264parse ! fdsink"
```